Extension/src/LanguageServer/configurations.ts

```diff
@@ -1260,7 +1263,7 @@ export class CppProperties {
     }
 
     private trimPathWhitespace(paths: string[] | undefined): string[] | undefined {
-        if (paths === undefined) {
+        if (paths === undefined || paths === null) {
             return undefined;
```

Comment:
According to the typings in the function signature, `paths` cannot be `null` here.